checks/plugins.py

```diff
@@ -36,7 +36,7 @@ def checkPluginList(lines):
         thirdPartyPlugins = []
 
         for s in pluginList:
-            if ('     ' in s):
+            if ('     ' in s and s.count(':') == 3):
```

Comment:
I wrote a review comment and then accidentally closed the tab before submitting. I thought we had cases of timestamps sometimes being affected by locale, but it seems we fixed that for OBS Studio 27.2.0. Still, if I recall correctly some Linux logs (console/debug) do not use timestamps, but log severity labels (`debug: `, `info: `, `warning: `, `error: `), so I wonder if this is still safer:
```
if ('     ' in s and s.count(': ') == 2):
```

In practice, I'm not sure how many people run such logs through the analyzer, if any.

Reply:
I think it would be fine to check for 3 or less :'s instead, I wasn't aware with this being the case with some logs. I will modify later tonight.